.github/workflows/cicd-dependency-review.yml

```diff
@@ -1,9 +1,9 @@
 # Need a GitHub Advanced Security license to run this action on private repos.
 
-name: Dependency Review
+name: ♻️ Dependency Review
 on:
   pull_request:
-    types: [opened, edited, reopened, synchronize]
+    branch: main
```

Comment:
was this run on any branch previously?

Reply:
Yes 😁 But the previous syntax is a bit too lax, in the sense that you end up running CICD checks for things like the PR title changing 🏃